utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -79,6 +79,7 @@
 "BSD 3-clause New License": BSD-3-Clause
 "BSD 4 Clause": BSD-4-Clause
 "BSD Four Clause License": BSD-4-Clause
+"BSD License": BSD-3-Clause
```

Comment:
We deliberately removed this in 391fe7144d05e7c41e2040ca4aa72ad6841c15f6 as it depends on the context / concrete package whether "BSD License" actually mean 3-Clause or 2-Clause; so we cannot have a generic mapping for it here.

I acknowledge there are probably more cases on non-generic mapping in this file which should be removed eventually, but we should not add more of these.

Reply:
Could be contributed to the 'ort-config' repo...

Reply:
> Could be contributed to the 'ort-config' repo...

As a concluded license for that particular package, yes, maybe.

Reply:
If you agree I will create a commit to remove the other ambiguous mappings.

Reply:
> we should remove all of them immediately and not only prevent new ones from being added.

I agree. So far just a lack of time / priorities and the fear that such a PR would break too many things for our users prevented us from doing it. But if the consensus now is to do it, I can prepare a PR.

Reply:
> I can prepare a PR.

Your comment and mine overlapped. Yes, please do prepare a PR.

Reply:
> Rather as a declared license mapping for that package, concluding a license based only on the declared license should not be a default.

+1